cmd/rancherd/go.mod

```diff
@@ -76,6 +76,7 @@ require (
 	github.com/rancher/k3s v1.21.1-rc1.0.20210514205428-fd4d226e3deb
 	github.com/rancher/rke2 v1.20.8-0.20210526012514-ea6514775f81 // v1.20.7+rke2r2 ... If you update this, also update RKE2_VERSION in rancher/package/Dockerfile.runtime
 	github.com/rancher/wrangler v0.6.2
+	github.com/satori/go.uuid v1.2.1-0.20181016170032-d91630c85102 // indirect
```

Comment:
https://ubuntu.com/security/CVE-2021-3538
Is this avoid that ?  
> from commit 0ef6afb2f6cdd6cdaeee3885a95099c63f18fc8c to d91630c8510268e75203009fe7daf2b8e1d60c45.

and this is landing right on d916, so I guess it depends on the word "from -> to"  Does trivy say this commit is clean but former is not ?

Reply:
This is the exact version that trivy recommends as secure. This commit also lands after the commit 75cca531ea763666bc46e531da3b4c3b95f64557 which fixed the issue https://github.com/satori/go.uuid/commits/master